#!/usr/bin/env python
PACKAGE = "dynamic_tutorials"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("kp_x", double_t, 0, "kp_x",    .02, 0,   0.1)
gen.add("kd_x", double_t, 0, "kp_x",    .01, 0,   0.1)
gen.add("kp_y", double_t, 0, "kp_y",    .02, 0,   0.1)
gen.add("kd_y", double_t, 0, "kd_y",    .01, 0,   0.1)

exit(gen.generate(PACKAGE, "face_controller", "configure_pid"))
